apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -192,29 +225,27 @@ class FilterBar extends React.Component {
               ))
             }
           </FilterDropDown>
-
+          { /*
           <FilterDropDown caption="Status" enabled={false}>
           </FilterDropDown>
           <FilterDropDown caption="Deadline" enabled={false}>
           </FilterDropDown>
           <FilterDropDown caption="Experience" enabled={false}>
           </FilterDropDown>
+           */ }
 
         </Overflow>
 
         <FilterDropDown caption="Sort by" enabled={true}>
-          <FilterMenuItem key={'1'} onClick={this.noop} style={{ display: 'flex', alignItems: 'flex-start' }}>
-            <div><CheckButton onChange={this.noop} checked={false} /></div>
-            <ActionLabel> ...label </ActionLabel>
-          </FilterMenuItem>
-          <FilterMenuItem key={'2'} onClick={this.noop} style={{ display: 'flex', alignItems: 'flex-start' }}>
-            <div><CheckButton onChange={this.noop} checked={false} /></div>
-            <ActionLabel>...deadline</ActionLabel>
-          </FilterMenuItem>
-          <FilterMenuItem key={'3'} onClick={this.noop} style={{ display: 'flex', alignItems: 'flex-start' }}>
-            <div><CheckButton onChange={this.noop} checked={false} /></div>
-            <ActionLabel>status</ActionLabel>
-          </FilterMenuItem>
+          { this.state.sortBy.map(s =>
```

Comment:
This should be calculated outside the return and captured into a variable instead

Reply:
Discussed, left for refactoring in the nearest future.